components/Home/sidebar/SidebarItems/BountiesSidebarItem.tsx

```diff
@@ -0,0 +1,112 @@
+import { css, StyleSheet } from "aphrodite";
+import { ID, NullableString } from "~/config/types/root_types";
+import { ReactElement } from "react";
+import AuthorFacePile from "~/components/shared/AuthorFacePile";
+import colors from "~/config/themes/colors";
+
+type Props = {
+  bountyAmount: number;
+  bountyContentSnippet: string;
+  createdByAuthor: any;
+  expirationDate: string;
+  relatedDocID: ID;
+  slug: NullableString;
+};
+export default function BountiesSidebarItem({
+  bountyAmount,
```

Comment:
This component should really just take in a single `Bounty` object with a couple of extra things.
Feel free to modify the `Bounty` object

Reply:
same here.